lib/jsdom/living/child-node.js

```diff
@@ -0,0 +1,91 @@
+"use strict";
+
+module.exports = function(core) {
+  // https://dom.spec.whatwg.org/#interface-childnode
+  // NoInterfaceObject
+
+  function ensureNodeArguments(doc, args) {
+    var nodes = [];
+    for(var i=0; args.length > i; i++) {
+      var arg = args[i];
+      if(typeof arg == "string") {
+        arg = doc.createTextNode(arg);
+      }
+
+      nodes.push(arg);
+    }
+
+    return nodes;
+  }
+
+  var ChildNodeInterface = {
+
+    // Inserts nodes just before node, while replacing strings in nodes with equivalent Text nodes.
+    // https://dom.spec.whatwg.org/#dom-childnode-before
+    before: function() {
+      if(this.parentNode !== null) {
+
+        var nodes = ensureNodeArguments(this._ownerDocument, arguments);
+
+        for(var i=0; nodes.length > i; i++) {
+          var node = nodes[i];
+          this.parentNode.insertBefore(node, this);
+        }
+      }
+    },
+
+    // Inserts nodes just after node, while replacing strings in nodes with equivalent Text nodes.
+    // https://dom.spec.whatwg.org/#dom-childnode-after
+    after: function() {
+      if(this.parentNode !== null) {
+
+        var nodes = ensureNodeArguments(this._ownerDocument, arguments);
+        var nextSibling = this.nextSibling;
```

Comment:
See https://dom.spec.whatwg.org/#dom-childnode-replacewith. It's the first following sibling *not in _nodes_*.